Yodel/Functions/MsSql/Initialize-YMsSqlDatabase.ps1

```diff
@@ -0,0 +1,43 @@
+
+function Initialize-YMsSqlDatabase
+{
+    <#
+    .SYNOPSIS
+    Esnures a Microsoft SQL Server database exists.
+
+    .DESCRIPTION
+    The `Initialize-YMsSqlDatabase` function creates a database in Microsoft SQL Server if it doesn't already exist.
+    Pass the connection to SQL Server to the `Connection` parameter and the name of the database to the `Name`
+    parameter. If the database doesn't exist (i.e. there's no record for it in `sys.databases`), it is created.

```

Comment:
From these docs, I can't figure out how I would know if the function actually did anything or not. E.g. is anything returned? A message written?